helm/charts/config/templates/load-init-config.yml

```diff
@@ -0,0 +1,40 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "config.fullname" . }}
+spec:
+  template:
+    metadata:
+      name: {{ template "gluu.name" . }}-job
+      labels:
+        app: {{ template "gluu.name" . }}-load
+    spec:
+      volumes:
+        - name: {{ template "gluu.name" . }}-config
+          persistentVolumeClaim:
+            claimName: {{ include "config.fullname" . }}
+        - name: {{ include "config.fullname" . }}-mount-gen-file
+          configMap:
+            name: {{ include "config.fullname" . }}-gen-json-file
+        - name: {{ include "config.fullname" . }}-tls-script
+          configMap:
+            name: {{ include "config.fullname" . }}-tls-script
+      containers:
+      - name: {{ template "gluu.name" . }}-load
+        image: gluufederation/config-init:4.0.0_dev
```

Comment:
Please change this to a flexible variable. The tag is what should be flexible. We should be able to allow installations for different versions of gluu.
'image: gluufederation/config-init:{{ .Values.configTag }}'